src/sidebar/store/modules/annotations.ts

```diff
@@ -303,61 +301,21 @@ function addAnnotations(annotations: Annotation[]) {
     dispatch: Dispatch,
     getState: () => {
       annotations: State;
-      route: RouteState;
       session: SessionState;
     }
   ) {
-    const added = annotations.filter(annot => {
-      return (
-        !annot.id || !findByID(getState().annotations.annotations, annot.id)
-      );
-    });
-
     const profile = sessionModule.selectors.profile(getState().session);
 
-    dispatch(
-      makeAction(reducers, 'ADD_ANNOTATIONS', {
-        annotations,
-        currentAnnotationCount: getState().annotations.annotations.length,
-        currentUserId: profile.userid,
-      })
-    );
-
-    // If we're not in the sidebar, we're done here.
-    // FIXME Split the annotation-adding from the anchoring code; possibly
-    // move into service
-    if (routeModule.selectors.route(getState().route) !== 'sidebar') {
-      return;
-    }
+    // The `ADD_ANNOTATIONS` action payload is defined in this module but there
+    // is an `ADD_ANNOTATIONS` reducer in the `selection` store module that
+    // relies on the `currentAnnotationCount` prop.
```

Comment:
This cost me about 25 minutes to track down. I was this close to removing the `currentAnnotationCount` property here, thinking it unused.